rovilay/phone_number-generator
# Generate phone numbers

#### What does this PR do?
This PR covers the tasks outlined below 👇🏽

#### Description of Task to be completed?
    * setup routes
    * add error handler middleware
    * setup cors

#### How should this be manually tested?
- git clone this app - `git clone https://github.com/rovilay/phone_number-generator.git`
- navigate to the cloned app
- git checkout to this branch - `git checkout -b ft-generate-phone-numbers`
- set up the `.env` with the `.env.sample` file
- install dependencies - `yarn install`
- start the app - `yarn run server`
- navigate to `localhost:port/api/v1/phonenumbers/generate` on the browser
- you should get a list of `phone numbers`

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
N/A

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
